### PR TITLE
Added calculator,word_count

### PR DESCRIPTION
**Pull request by Kousalya1205**

### CODE REVIEW

**Review by harsha-sridhar — COMMENTED** [review]
Change the directory structure
src/
     data/
      - sampletextfile.txt
   - prog1.py
   - prog2.py
test/

File: word_count.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python
+import sys
+try:
+    n=len(sys.argv)
+    if n==2:
+        fileObject = open(sys.argv[1], "r")
+        str1=fileObject.read()
```
Comment (harsha-sridhar) [inline word_count.py]: Explore the usage of readlines()

File: word_count.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        num_lines=0
+        num_char=0
+        num_words=len(str1.split())
+        for i in str1:
```
Comment (harsha-sridhar) [inline word_count.py]: Optimization required

File: calculator.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+def multiply (num1,num2):
+    return num1*num2
+def divide (num1,num2):
+    return num1/num2
+try:
+    number1=int(input("enter the number :"))
+    operations=['+','-','*','/']
+    to_continue='y'
+    result=0
+    while to_continue == 'y':
```
Comment (harsha-sridhar) [inline calculator.py]: Optimization required

**Comment by Kousalya1205** [comment]
>

File: calculator.py (hunk 3 above)
Comment (Kousalya1205) [reply calculator.py]: Code Optimization is done

File: src/word_count.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python
+import sys
+try:
+    n=len(sys.argv)
+    if n==2:
+        fileObject = open(sys.argv[1], "r")
+        str1=fileObject.read()
+        num_lines=0
+        num_char=0
+        num_words=len(str1.split())
```
Comment (harsha-sridhar) [inline src/word_count.py]: Explore the usage of readlines()

File: word_count.py (hunk 1 above)
Comment (Kousalya1205) [reply word_count.py]: modified with readlines() method

**Review by harsha-sridhar — APPROVED** [review]
PR Approved

**Pull request merged**